webapp-django/crashstats/crashstats/templates/crashstats/crashes_per_user.html

```diff
@@ -0,0 +1,275 @@
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.structure.css') }}" media="screen" />
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.theme.css') }}" media="screen" />
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/metricsgraphics.css') }}">
+      <link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/metricsgraphics_custom.css') }}">
+  {% endcompress %}
+  {% compress css %}
+      <link rel="stylesheet" type="text/less" href="{{ static('crashstats/css/crashes_per_user.less') }}">
+  {% endcompress %}
+{% endblock %}
+
+{% block page_title %}
+Crashes per User for {{ product }}
+{% endblock %}
+
+{% block content %}
+<div id="mainbody">
+
+<div class="page-heading">
+    <p style="float: right">
+      <a href="{{ old_daily_report_url }}">View the old <i>Crashes per user</i> report</a>
+    </p>
+    <h2>Crashes per User for {{ product }}</h2>
+</div>
+
+
+<div class="panel daily_search">
+    <div class="title daily_search_title">
+        <h2>Options</h2>
+    </div>
+
+    <div class="body daily_search_body">
+        <div id="daily_search">{# is this div tag necessary? #}
+        <div class="daily_search_version">
+            <form name="daily_search_version_form" action="">
+                <p>
+                  All ADI ratios are generated per 100 ADIs, not per single ADI.
+                  "3.5 crashes/ADI" means that there are 3.5 crashes per 100
+                  installations, not per installation. This ratio also does
+                  not distinguish between installs who crash multiple times
+                  and multiple crashing installs.
+                </p>
+
+                <table class="by_version">
+                    <tr>
+                        <th>Product</th>
+                        <td>
+                          <label class="visually-hidden" for="daily_search_version_form_products">Select Product</label>
+                        <select id="daily_search_version_form_products" name="p">
+                          {% for product_name in releases %}
+                            <option value="{{ product_name }}" {% if product == product_name %}selected{% endif %}>{{ product_name }}</option>
+                            {% endfor %}
+                        </select>
+                        </td>
+                    </tr>
+
+                    {% for i in range(4)|reverse %}
+                    <tr>
+                        {% if loop.first %}
+                        <th rowspan="4">Versions</th>
+                        {% endif %}
+
+                        <td>
+                          <label class="visually-hidden" for="version{{ i }}">Select Version</label>
+                            <select id="version{{ i }}" name="v">
+                                <option value="">-- versions --</option>
```

Comment:
Should this value be disabled? Or could it be replaced with a placeholder, even? (Maybe that's a bit out of scope... )